spec/bitfields_spec.rb

```diff
@@ -539,7 +539,13 @@ class ManyBitsUser < User
 
     describe 'with OR' do
       it "generates sql for each bit" do
-        User.bitfield_sql({:seller => true, :insane => true, :stupid => false}, :query_mode => :bit_operator_or).should == '(users.bits & 1) = 1 OR (users.bits & 2) = 2 OR (users.bits & 4) = 0'
+        User.bitfield_sql({:seller => true, :insane => true, :stupid => false}, :query_mode => :bit_operator_or).should == '(users.bits & 3) <> 0 OR (users.bits & 4) <> 4'
+      end
+      it "generates sql for only ON" do
+        User.bitfield_sql({:seller => true, :insane => true}, :query_mode => :bit_operator_or).should == '(users.bits & 3) <> 0'
+      end
+      it "generates sql for only OFF" do
+        User.bitfield_sql({:seller => false, :stupid => false}, :query_mode => :bit_operator_or).should == '(users.bits & 5) <> 5'
       end
```

Comment:
make sure there are tests for "multiple on" and "multiple off" bits so we make sure this actually works :D

Reply:
tests added